### PR TITLE
merge RST upstream into clutch branch

### PR DESCRIPTION
**Pull request by smonero**
There has been a lot of divergence between the forks so we need this to keep things in a clean state

### CODE REVIEW

File: src/Timeline.tsx [hunk 1]
```diff
@@ -96,274 +81,141 @@ export const Timeline = <EID extends string, LID extends string, E extends Timel
...
-                        />
-                      )}
-                    </g>
-                  )
-                }}
-              </InteractionHandling>
-            )
-          }}
-        </MouseAwareSvg>
+        <svg viewBox={`0 0 ${width} ${height}`} width={width} height={height} style={{ outlineStyle: 'none'}}>
```
Comment (smonero) [inline src/Timeline.tsx:212]: I am not sure this is the right place to put this - but when I don't put it then we get the blue box selection problem

**Review by dschaller — APPROVED** [review]
This is a bit tough because the commit history will be a bit off but I think that's ok

**Comment by dschaller** [comment]
Triggering CLA

**Comment by dschaller** [comment]
Ah right this is an upstream update so it won't pass since there are authors that won't have signed the CLA. I'll manually merge into our branch

**Pull request merged**